app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -16,7 +16,7 @@
   -->
 
 <resources>
-    <!-- MAINTAINED BY: (@sr093906) -->
+    <!-- 维护者： (@sr093906) -->
```

Comment:
thank you, but this line as to stay english for the language badge script.

The needed key is `gallery_all_photos_label`